gigareview/console/tests/acceptance/EMReportJob.feature

```diff
@@ -0,0 +1,31 @@
+Feature:
+  As an editor
+  I want the automatically downloaded EM reports spreadsheet to be ingested automatically
+  So that I can maintain an up-to-date database of peer reviews to be curated and to be shown on GigaReviews
+
+  Background:
+    Given the database is reset
+
+  @ok
+  Scenario: EM manuscript report is downloaded and saved to manuscript table
+    Given the file is on the sftp server
```

Comment:
The `theFileIsOnTheSftpServer()` function for this step checks if all the EM report files are available on the test SFTP server, not just one file. Might be better to say `Given EM report files are on the sftp server` in this statement.